src/App.js

```diff
@@ -46,16 +33,12 @@ const updatedSearch = e =>{
         <button className="search-button" type="submit"> Search Park </button>
       </form>
 
-  <div className="park-description">
-    {data.map(fullName =>(
-      <Park
-      title ={data.data}
-      />
-      
-  ))}
-
+      <div className="park-description">
+        {parks.map(park => (
+          <Park data={park} />
```

Comment:
I figure it might be easiest while you're still messing around with the app to just pass in all the park data as a property called `data`